### PR TITLE
Fixed login functionality, added logout functionality

### PR DESCRIPTION
**Pull request by lili2311**

### CODE REVIEW

File: pimpmycause/pimpmycause/urls.py [hunk 1]
```diff
@@ -17,16 +17,16 @@
...
 from core.views import HomepageView, ContactView, AboutView, MeetTheTeam, OurPartners, Partnerships, BecomeAPartner, WhoWeAre, HowItWorks
-from profiles.views import LoginView, RegistrationView, RegistrationComplete, ActivationComplete, profile_update
+from profiles.views import RegistrationView, RegistrationComplete, ActivationComplete, profile_update, logout_view
 from profiles.forms import PimpUserRegistrationForm
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^$', HomepageView.as_view(), name="index"),
-    url(r'^login/', LoginView.as_view(), name="login"),
+    url(r'^logout/', logout_view, name="logout_view"),
```
Comment (yanabar) [inline pimpmycause/pimpmycause/urls.py]: Maybe this should be like : url(r'^logout/', LogoutView.as_view(), name="logout"),
to match the others? If so L20 above needs updating and L39 from views.py.

Comment (lili2311) [inline pimpmycause/pimpmycause/urls.py]: The logout view is different from others it is not a Class Based view but it is actually a function based view like profile_page, I am following Pep8 convention as advised by Django docs: https://www.python.org/dev/peps/pep-0008/#function-names and https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/

Comment (yanabar) [inline pimpmycause/pimpmycause/urls.py]: i see :)

Comment (leungant) [inline pimpmycause/pimpmycause/urls.py]: nice!

**Pull request merged**